src/util/const.ts

```diff
@@ -84,6 +88,14 @@ export const CustomNetworks: { [key: string]: NetworkFromEthers } = {
   'polygonzkevm-testnet': {
     chainId: 1442,
     name: 'polygonzkevm-testnet'
+  },
+  'base-mainnet': {
+    chainId: 8453,
+    name: 'polygonzkevm-mainnet'
```

Comment:
update the name fields here and below

Reply:
ah sorry, weird thought this might've failed unit tests